### DISCUSSION TITLE
Regarding coding environment

### DISCUSSION THREAD

**Discussion by Dark-Base** [community/community#68995] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Suggest me any alternates of VSCode editor

**Comment by RitamPaul** [comment]
sublime text editor, atom, Eclipse, Dev-CPP, any many more

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
